client/src/components/hooks/useConfiguration.js

```diff
@@ -0,0 +1,16 @@
+import { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { getNetwork } from '@cd/selectors/settings';
+import { getConfigKey } from '@cd/services/configurationServices';
+
+export const useConfiguration = () => {
```

Comment:
I still can see some places that is using `getConfigKey` , why didn't we refactor all ?

Reply:
Hi @djhahe , I pushed commit (d1ae49bbaa81bc13800bbb81017207a98033e35a) to resolve this